### DISCUSSION TITLE
I wonder why my request gets always rejected

### DISCUSSION THREAD

**Discussion by byPixelTV** [community/community#77977] [GitHub Education]
### Hi everyone,

my request for the Github Student Developer Pack gets always rejected an I don't know why! I'm at least 13 years old, I provided my private email because my school is blocking emails from other mailservers, and I took a picture of my student ID with my device camera using Iriun Webcam because I'm using a desktop PC without a webcam! I don't know if it is a problem that my student ID is in German because I'm from germany! And I added an screenshot that my school is blocking mails from other mailservers
![Screenshot 2023-12-05 223630](https://github.com/community/community/assets/113579672/0c40edfc-6481-47ac-9d79-fdfddfeb2599) ![Screenshot 2023-12-05 223929](https://github.com/community/community/assets/113579672/1527253a-55dd-4375-b666-f65d5cdf2c0c)

**Comment by IgnacioPalma** [comment]
Hello @byPixelTV! :wave: 

Did your verification documentation clearly display your **name**, the **academic institution** you are part of, and a **date**? :thinking:

**Reply by byPixelTV** [reply]
On my student ID is a date, my first and last name, and the name of my school
